pyboy/plugins/game_wrapper_super_mario_land.py

```diff
@@ -21,6 +22,140 @@
 except ImportError:
     cythonmode = False
 
+tiles_minimal = np.zeros((24, 16), dtype=np.uint8)
```

Comment:
I think a lot of the assignments below could be parameterized/automated. There's only hardcoded values. Would it improve readability as well, if it was structure as a for-loop and the ids were automatically incremented?

Reply:
Yeah, I know it was a first shot, and I wanted feedback on the tiles_compressed and tiles_minimal before investing some more times !

I'll try to think to a cleaver way now ^^

Reply:
Done